docs/bugzilla

```diff
@@ -21,6 +21,10 @@ branch: you probably want to set this to "master".
 If the central repository option is enabled and there is a "fix",
 "close", or "address" before the bug then that bug is closed.
 
+On bugzilla server side you should disable option `commentonchange_resolution`.
+With this option enabled - setting bugs to `RESOLVED FIXED` status will fail with message 
+`You have to specify a comment when changing the Resolution of a bug from (empty) to FIXED`
```

Comment:
`Without this option disabled when X sets bugs to the RESOLVED FIXED status this will fail with the message You have to specify a comment when changing the Resolution of a bug from (empty) to FIXED`